### PR TITLE
Add chat

### PR DESCRIPTION
**Pull request by audunste**

### CODE REVIEW

**Comment by audunste** [comment]
We can have discussions here

File: plan.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ Write plans and concepts
   - Simplify many Behaviors to simply wrap one action
   - We should be able to ask Behaviors if they can fulfil a requirement (a need, inventory status, environmental status)
   - Behaviors can then recursively use each other to create more complex behavior
-  
\ No newline at end of file
+  
+  
+- Have chat
```
Comment (olavemil) [inline plan.md:9]: We could also make use of the wiki functionality github provides to keep track of current functionality/planned features.

**Comment by audunste** [comment]
Hah, I forgot about the wiki functionality even if I talked about it myself when robin and I discussed this yesterday. Plan.md should be in the wiki, if the wiki functionality is good.

This pull request is good for chat/forum type discussions that the wiki probably does not have.

**Comment by olavemil** [comment]
Agreed, wiki would be in addition, for example to store the result of discussion. Conversation history is usually a poor way to store information for retrieval efficiency.

**Comment by olavemil** [comment]
Working on separating attributes, needs skills and health into separate components, since they behave differently and to avoid cluttering the attribute namespace.

My current plan for allowed dependencies is 
attributes(none)
health(attributes)
skills(attributes, ?health?)
needs(attributes, health, skills).

It makes sense for needs to be base of all of these, or at least attributes and health, but do we need to allow skills to be adjusted by health? Health would likely not need access to either needs or skills...

**Comment by olavemil** [comment]
On another note, at first I thought this was a pull request for a (slightly premature, due to the lack of multiplayer,) chat feature in-game, or an ability for the agents to have a chat.

**Comment by audunste** [comment]
Hehe, well both of those would be nice, but I'm not making any of those at the moment. Currently I'm making logging more modular, so that it can be enabled for individual agents. I needed if for understanding how the attributes change

**Pull request closed without merging**